src/tests/ftest/erasurecode/restart.py

```diff
@@ -47,6 +47,13 @@ def execution(self, agg_check=None):
             time.sleep(20)
 
         if agg_check == "Before":
+            # Over-write all EC object data to NVMe
+            self.ior_write_dataset(operation="Auto_Write", percent=self.percent)
+            self.log.info(self.pool.pool_percentage_used())
+            # Over-write all EC object data to SCM
+            self.ior_write_dataset(storage='SCM', operation="Auto_Write", percent=self.percent)
+            self.log.info(self.pool.pool_percentage_used())
+            self.pool.pool_percentage_used()
```

Comment:
From the ticket:
> Per Jeff’s comment about aggregation on writing data to storage/pool:
>
> 1. Large writes with no overwrite or file deletion - should not see any aggregation
>
> 2. Large writes with overwrite - should see reduction in SCM and NVMe usage after aggregation
>
> 3. Small contiguous writes - should see reduction in SCM usage and increase in NVMe usage after aggregation

Couldn't we just delete IOR files instead of overwriting?